lib/utils/tests-options-validator.js

```diff
@@ -48,7 +48,49 @@ Object.defineProperty(TestsOptionsValidator.prototype, 'shouldSplit', {
  */
 function validatePartitions(partitions, split) {
   validatePartitionSplit(partitions, split);
-  validatePartitionsUnique(partitions);
+  validateElementsUnique(partitions);
+}
+
+/**
+ * Validates the specified reply-browser
+ *
+ * @param {*} replayBrowser
+ * @param {*} replayExecution
+ */
+function validateReplyBrowser(replayBrowser, replayExecution) {
+  if(!replayExecution) {
+    throw new Error('You must specifiy a \'reply-execution\' value in order to use \'reply-browser\' option.');
+  }
+
+  for (const i in replayBrowser) {
+    const browserId = replayBrowser[i]
+    if (browserId < 1) {
+      throw new Error('You must specify reply-browser values greater than or equal to 1.');
+    }
+  }
+
+  validateElementsUnique(replayBrowser);
+}
+
+/**
+ * Validates the specified reply-browser
+ *
+ * @param {*} replayBrowser
+ * @param {*} replayExecution
+ */
+function validateReplyBrowser(replayBrowser, replayExecution) {
+  if(!replayExecution) {
+    throw new Error('You must specifiy a \'reply-execution\' value in order to use \'reply-browser\' option.');
+  }
+
+  for (const i in replayBrowser) {
+    const browserId = replayBrowser[i]
```

Comment:
Same comment about not needing a separate constant - just use replayBrowser[i] in the if statement.